tests/unit/plugins/test_dotnet.py

```diff
@@ -95,6 +95,12 @@ def side_effect(cmd, *args, **kwargs):
 
 class DotNetProjectTestCase(DotNetProjectBaseTestCase):
 
+    def test_sdk_in_path(self):
+        plugin = dotnet.DotNetPlugin(
+            'test-part', self.options, self.project)
+        self.assertThat(plugin.env(plugin.installdir), Contains(
+            'PATH={}:$PATH'.format(plugin._dotnet_sdk_dir)))
```

Comment:
If we keep the "only set path if root is installdir" then we should probably test that scenario as well.